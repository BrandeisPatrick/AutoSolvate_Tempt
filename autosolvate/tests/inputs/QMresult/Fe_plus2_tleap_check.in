source oldff/leaprc.ff14SB
source leaprc.gaff
source leaprc.water.opc
addAtomTypes {
        { "M1"  "Fe" "sp3" }
        { "Y7"  "N" "sp3" }
        { "Y8"  "N" "sp3" }
        { "Y9"  "N" "sp3" }
        { "Z1"  "N" "sp3" }
        { "Z2"  "N" "sp3" }
        { "Z3"  "N" "sp3" }
}
FE1 = loadmol2 FE1.mol2
L01 = loadmol2 L01.mol2
L11 = loadmol2 L11.mol2
L21 = loadmol2 L21.mol2
loadamberparams LG0.frcmod
loadamberparams LG1.frcmod
loadamberparams LG2.frcmod
loadamberparams frcmod.ionslm_126_opc
loadamberparams Fe_plus2_mcpbpy.frcmod
mol = loadpdb Fe_plus2_mcpbpy.pdb
bond mol.1.FE mol.2.N
bond mol.1.FE mol.2.N1
bond mol.1.FE mol.3.N
bond mol.1.FE mol.3.N1
bond mol.1.FE mol.4.N
bond mol.1.FE mol.4.N1
savepdb mol Fe_plus2_dry.pdb
saveamberparm mol Fe_plus2_dry.prmtop Fe_plus2_dry.inpcrd
solvatebox mol OPCBOX 10.0
addions mol Na+ 0
addions mol Cl- 0
savepdb mol Fe_plus2_solv.pdb
saveamberparm mol Fe_plus2_solv.prmtop Fe_plus2_solv.inpcrd
quit
 
