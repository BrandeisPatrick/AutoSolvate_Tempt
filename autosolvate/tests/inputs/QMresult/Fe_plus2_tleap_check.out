-I: Adding /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/prep to search path.
-I: Adding /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib to search path.
-I: Adding /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/parm to search path.
-I: Adding /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/cmd to search path.
-f: Source Fe_plus2_tleap_check.in.

Welcome to LEaP!
(no leaprc in search path)
Sourcing: ./Fe_plus2_tleap_check.in
----- Source: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/cmd/oldff/leaprc.ff14SB
----- Source of /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/cmd/oldff/leaprc.ff14SB done
Log file: ./leap.log
Loading parameters: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/parm/parm10.dat
Reading title:
PARM99 + frcmod.ff99SB + frcmod.parmbsc0 + OL3 for RNA
Loading parameters: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/parm/frcmod.ff14SB
Reading force field modification type file (frcmod)
Reading title:
ff14SB protein backbone and sidechain parameters
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/amino12.lib
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/aminoct12.lib
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/aminont12.lib
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/nucleic12.lib
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/atomic_ions.lib
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/solvents.lib
----- Source: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/cmd/leaprc.gaff
----- Source of /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/cmd/leaprc.gaff done
Log file: ./leap.log
Loading parameters: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/parm/gaff.dat
Reading title:
AMBER General Force Field for organic molecules (Version 1.81, May 2017)
----- Source: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/cmd/leaprc.water.opc
----- Source of /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/cmd/leaprc.water.opc done
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/atomic_ions.lib
Loading library: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/lib/solvents.lib
Loading parameters: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/parm/frcmod.opc
Reading force field modification type file (frcmod)
Reading title:
Parameters for OPC water (JPCL, 2014, 5 (21), pp 3863-3871)
Loading parameters: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/parm/frcmod.ionslm_126_opc
Reading force field modification type file (frcmod)
Reading title:
Li/Merz ion parameters of atomic ions for the OPC water model (12-6 set)
Loading Mol2 file: ./FE1.mol2
Reading MOLECULE named FE1
Loading Mol2 file: ./L01.mol2
Reading MOLECULE named L01
Loading Mol2 file: ./L11.mol2
Reading MOLECULE named L11
Loading Mol2 file: ./L21.mol2
Reading MOLECULE named L21
Loading parameters: ./LG0.frcmod
Reading force field modification type file (frcmod)
Reading title:
Remark line goes here
Loading parameters: ./LG1.frcmod
Reading force field modification type file (frcmod)
Reading title:
Remark line goes here
Loading parameters: ./LG2.frcmod
Reading force field modification type file (frcmod)
Reading title:
Remark line goes here
Loading parameters: /home/sunnyxun/anaconda3/envs/autosolvate/dat/leap/parm/frcmod.ionslm_126_opc
Reading force field modification type file (frcmod)
Reading title:
Li/Merz ion parameters of atomic ions for the OPC water model (12-6 set)
Loading parameters: ./Fe_plus2_mcpbpy.frcmod
Reading force field modification type file (frcmod)
Reading title:
REMARK GOES HERE, THIS FILE IS GENERATED BY MCPB.PY
Loading PDB file: ./Fe_plus2_mcpbpy.pdb
  total atoms in file: 61
Writing pdb file: Fe_plus2_dry.pdb
Checking Unit.

/home/sunnyxun/anaconda3/envs/autosolvate/bin/teLeap: Warning!
The unperturbed charge of the unit (2.000000) is not zero.

/home/sunnyxun/anaconda3/envs/autosolvate/bin/teLeap: Note.
Ignoring the warning from Unit Checking.

Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
 total 30 improper torsions applied
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
  (Residues lacking connect0/connect1 - 
   these don't have chain types marked:

	res	total affected

	L01	1
	L11	1
	L21	1
  )
 (no restraints)
  Solute vdw bounding box:              13.848 12.754 11.036
  Total bounding box for atom centers:  33.848 32.754 31.036
  Solvent unit box:                     18.865 18.478 19.006
  Total vdw box size:                   36.600 36.022 34.203 angstroms.
  Volume: 45093.627 A^3 
  Total mass 18270.162 amu,  Density 0.673 g/cc
  Added 985 residues.

/home/sunnyxun/anaconda3/envs/autosolvate/bin/teLeap: Warning!
addIons: 1st Ion & target unit have charges of the same sign:
     unit charge = 2; ion1 charge = 1;
     can't neutralize.
2 Cl- ions required to neutralize.
Adding 2 counter ions to "mol" using 1A grid
Grid extends from solute vdw + 2.36  to  8.27
Resolution:      1.00 Angstrom.
Solvent present: replacing closest with ion
	 when steric overlaps occur
Calculating grid charges
(No solvent overlap)
Placed Cl- in mol at (-3.65, 4.86, 1.87).
(Replacing solvent molecule)
Placed Cl- in mol at (5.30, -6.41, -3.06).

Done adding ions.
Writing pdb file: Fe_plus2_solv.pdb
   printing CRYST1 record to PDB file with box info
Checking Unit.

/home/sunnyxun/anaconda3/envs/autosolvate/bin/teLeap: Note.
Ignoring the warnings from Unit Checking.

Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
 total 30 improper torsions applied
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
  (Residues lacking connect0/connect1 - 
   these don't have chain types marked:

	res	total affected

	L01	1
	L11	1
	L21	1
  )
 (no restraints)
	Quit

Exiting LEaP: Errors = 0; Warnings = 2; Notes = 2.
